kore/test/Test/Kore/Unification/SubstitutionNormalization.hs

```diff
@@ -296,6 +296,24 @@ test_normalize =
                 , denormalized = []
                 }
         ]
+    , testGroup "deduplication"
+        [ test "produces substitution"
+            [(t, mkVar x), (t, mkVar y)]
```

Comment:
I realized `test` takes a `Map` not a list, so this gets automatically converted to a singleton (t, y)